client/src/plus/updates/index.tsx

```diff
@@ -0,0 +1,237 @@
+import Container from "../../ui/atoms/container";
+
+import { useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import useSWR from "swr";
+import { DocMetadata } from "../../../../libs/types/document";
+import { MDN_PLUS_TITLE } from "../../constants";
+import BrowserCompatibilityTable from "../../document/ingredients/browser-compatibility-table";
+import { browserToIconName } from "../../document/ingredients/browser-compatibility-table/headers";
+import { useLocale, useScrollToTop } from "../../hooks";
+import { Button } from "../../ui/atoms/button";
+import { Icon } from "../../ui/atoms/icon";
+import { Loading } from "../../ui/atoms/loading";
+import Mandala from "../../ui/molecules/mandala";
+import { Paginator } from "../../ui/molecules/paginator";
+import BookmarkMenu from "../../ui/organisms/article-actions/bookmark-menu";
+import { NotificationsWatchMenu } from "../../ui/organisms/article-actions/notifications-watch-menu";
+import { useUserData } from "../../user-context";
+import { camelWrap, range } from "../../utils";
+import { Event, Group, useBCD, useUpdates } from "./api";
+import "./index.scss";
+
+const CATEGORY_TO_NAME = {
+  api: "Web APIs",
+  css: "CSS",
+  html: "HTML",
+  http: "HTTP",
+  javascript: "JavaScript",
+  mathml: "MathML",
+  svg: "SVG",
+  webdriver: "WebDriver",
+  webextensions: "Web Extensions",
+};
+
+export default function Updates() {
+  document.title = `Updates | ${MDN_PLUS_TITLE}`;
+  useScrollToTop();
+  const [searchParams] = useSearchParams();
+  const currentPage = parseInt(searchParams.get("page"), 10) || 0;
+  const { data } = useUpdates(currentPage);
+
+  return (
+    <div className="updates">
+      <header className="plus-header-mandala">
+        <Container>
+          <h1>
+            <div className="mandala-icon-wrapper">
+              <Mandala rotate={true} />
+              <Icon name="bell-filled" />
+            </div>
+            <span>Updates</span>
+          </h1>
+          <p>
+            Browse changes in our Browser Compatibility Data.
```

Comment:
From [MP-188](https://mozilla-hub.atlassian.net/browse/MP-188):

```suggestion
            Get the latest updates about browser compatibility of all the features in one place.
```